caffe2/opt/optimize_ideep.cc

```diff
@@ -21,6 +22,13 @@ void OptimizeForIdeep(repr::NNModule* nn) {
       return false;
     }
 
+    // IDEEP doesn't support fusion group conv
+    int group =
+        ArgumentHelper::GetSingleArgument<OperatorDef, int>(*op, "group", 1);
```

Comment:
There is `HasArgument`

Reply:
We still need to check whether it's 1 or not. If it's 1, it's ok. So GetSingleArgument here looks ok.